migrations/1660716115917-seedNotificationType.ts

```diff
@@ -1174,6 +1174,34 @@ export const GivethNotificationTypes = {
     ],
     content: 'You project {project name} is not verified anymore',
   },
+  PROJECT_UNVERIFIED_WHO_BOOSTED: {
+    name: 'Project unverified - Users Who Boosted',
+    microService: MICRO_SERVICES.givethio,
+    category: NOTIFICATION_CATEGORY.PROJECT_RELATED,
+    icon: '',
+    schemaValidator: SCHEMA_VALIDATORS_NAMES.PROJECT_UNVERIFIED_WHO_BOOSTED,
+    emailNotifierService: THIRD_PARTY_EMAIL_SERVICES.SEGMENT,
+    emailNotificationId: null,
```

Comment:
Add an email notification ID , and later Mitch or someone else will connect it to autopilot

Reply:
Only if this is sending emails with segment.

Reply:
Yeah you are right, I added it.